.github/scripts/mega-app-copy-files.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+cd build-system-tests
+if [ "$BUILD_TOOL" == 'cra' ];then
+    cp templates/template-aws-exports.js mega-apps/${MEGA_APP_NAME}/src/aws-exports.js
+    if [ "$LANGUAGE" == 'js' ];then
+        cp templates/components/react/cra/App.js mega-apps/${MEGA_APP_NAME}/src/App.js
+    else
+        cp templates/components/react/cra/App.js mega-apps/${MEGA_APP_NAME}/src/App.tsx
+        if [ "$FRAMEWORK_VERSION" == '16' ];then
+            cp templates/components/react/cra/index-react-${FRAMEWORK_VERSION}.js mega-apps/${MEGA_APP_NAME}/src/index.tsx
+        fi
+    fi
+fi
+
+if [ "$BUILD_TOOL" == 'next' ];then
+    mkdir mega-apps/${MEGA_APP_NAME}/data
+    cp templates/template-aws-exports.js mega-apps/${MEGA_APP_NAME}/data/aws-exports.js
+    cp templates/components/react/next/App.js mega-apps/${MEGA_APP_NAME}/pages/index.tsx
+    if [ "$BUILD_TOOL_VERSION" == '11' ];then
+        cp templates/components/react/next/template-package-${BUILD_TOOL_VERSION}.json mega-apps/${MEGA_APP_NAME}/package.json
+        cp templates/components/react/next/template-tsconfig-${BUILD_TOOL_VERSION}.json mega-apps/${MEGA_APP_NAME}/tsconfig.json
+    fi
```

Comment:
Same nit here, can we put a small comment here why this is necessary for 11?

Reply:
Done 3b8c87120cb30bf33034079b7895aff4854db685